jinahub/indexers/storage/RedisStorage/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM jinaai/jina:2.0.10 as base
+
+# install git
+RUN apt-get -y update && apt-get -y install redis-server git
+
+# install requirements before copying the workspace
+COPY requirements.txt /requirements.txt
+RUN pip install -r requirements.txt
+
+# setup the workspace
+COPY . /workspace
+
+WORKDIR /workspace
+
+FROM base as entrypoint
+ENTRYPOINT ["/bin/bash", "/workspace/scripts/entrypoint.sh"]
```

Comment:
We don't ship the services in the same Docker image. 

We only ship the executor in this Docker, and inform the users that they need to run Redis in another.

Also provide a one-liner to start a Redis Docker instance that works for this executor.

Check the PSQLStorage README https://github.com/jina-ai/executors/tree/main/jinahub/indexers/storage/PostgreSQLStorage#-prerequisites

Reply:
fixed